### DISCUSSION TITLE
Getting list of polygons from COCOEvaluator()

### DISCUSSION THREAD

**Discussion by wish2023** [facebookresearch/detectron2#2553] [Q&A]
When I run `evaluator = COCOEvaluator("sample_data", cfg, False, output_dir="./output/")` I get a file, `coco_instances_results.json` as expected. However, the json file looks like this:

`       "segmentation":{`
`          "size":[`
`             5482,`
`             7420`
`          ],`
`         "counts":"en_[e0ac0cg4\\4fK]2bM]1dNS1nN[1eNn0RO.............`

Is there a way to decode and convert the values of `segmentation` to a polygon format?

**Comment by hadilou** [comment]
Check  [pycoco api](https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/pycocotools/mask.py) _encode_ and _decode_ functions
